examples/pipe/dingo.ini

```
@@ -0,0 +1,129 @@
+# spline-calibration-nodes=10
+# spline-calibration-curves=1000
+
+
+################################################################################
+## Data generation arguments
+################################################################################
+
+# ignore-gwpy-data-quality-check=True
+# timeslide-file=None
+trigger-time= {{ production.meta['event time'] }} # GW150914 #
+# data-dict=None
+# data-format=None
+# allow-tape=None
+channel-dict = {H1:GWOSC, L1:GWOSC} #{ {% if production.meta['interferometers'] contains "H1" %}{{ production.meta['data']['channels']['H1'] }},{% endif %} {% if production.meta['interferometers'] contains "L1" %}{{ production.meta['data']['channels']['L1'] }},{% endif %}{% if production.meta['interferometers'] contains "V1" %}{{ production.meta['data']['channels']['V1'] }}{% endif %} }
+
+
+################################################################################
+## Detector arguments
+################################################################################
+
+detectors= [ {% if production.meta['interferometers'] contains "H1" %}H1{% endif %}{% if production.meta['interferometers'] contains "L1" %}, L1{% endif %}{% if production.meta['interferometers'] contains "V1" %}, V1{% endif %} ]
+# duration={{ production.meta['quality']['segment-length'] }}
+# psd-dict={ {% if production.meta['interferometers'] contains "H1" %}H1:{{ production.psds['H1'] }},{% endif %} {% if production.meta['interferometers'] contains "L1" %}L1:{{ production.psds['L1'] }},{% endif %} {% if production.meta['interferometers'] contains "V1" %}V1:{{ production.psds['V1'] }}{% endif %} }
+# psd-fractional-overlap=0.5
+# post-trigger-duration=2.0
+# sampling-frequency={{ production.meta['quality']['sample-rate'] }}
+psd-length= 128 #{{ production.meta['quality']['psd-length'] }}
+# psd-maximum-duration=1024
+# psd-method=median
+# psd-start-time=None
+# maximum-frequency=1024
+# minimum-frequency={{ production.meta['quality']['reference-frequency'] }}
+# tukey-roll-off=0.4
+# resampling-method=lal
+
+################################################################################
+## Injection arguments
+################################################################################
+
+# injection-dict=None
+# injection-file=None
+
+################################################################################
+##  Job submission arguments
+################################################################################
+
+accounting = ' ' # If running on a LIGO cluster, add accounting tag here
+# accounting-user = ""
+label={{ production.name }} # GW150914
+local = False
+# local-generation=False
+# local-plot=False
+outdir={{ production.rundir }} # outdir_GW150914
+#overwrite-outdir=True
+request-disk=5.0
+request-memory=32.0
+# request-memory-generation=None
+request-cpus=8
+request-cpus-importance-sampling = 64
+sampling-requirements = [TARGET.CUDAGlobalMemoryMb>40000]
+extra-lines = None
+simple-submission = True
```

Turn this off, since it will usually not run on MPI-IS cluster
```suggestion
simple-submission = False
```